react-exercise/src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './stylesheets/styles.scss'
+import Blogs from './app/blogs/Blogs';
```

Comment:
Khi import có cần theo thứ tự nào không vậy ạ? Hay là import kiểu gì cũng được.

Reply:
Keyword để search: import order reactjs **best practices**

Tham khảo thêm: https://betterprogramming.pub/21-best-practices-for-a-clean-react-project-df788a682fb